### PR TITLE
belindas-closet-nextjs_08_438_Fields_category_page_02

### PR DESCRIPTION
**Pull request by Diego-Cano**
This is a pull request!

Please fill out this description with the following:
Aims to Resolves [#438](https://github.com/SeattleColleges/belindas-closet-nextjs/issues/438#issue-2339438044)
I am trying to set up the logic for displaying only related fields for each product.  

Screenshot 2024-06-11 at 8 55 17 PM

### CODE REVIEW

**Review by alasali1 — COMMENTED** [review]
image
On my end it looks like all of the fields are missing rather than only irrelevant fields.
I would guess that most of the work to be done would be at the end starting in the <stack> element though I am not very strong in typescript. I would think that you might be able to add a line like `{product.productType === "Shoes" && (` to try and validate/compare the products productType property.
image

**Comment by kaimanasse15** [comment]
Works great!
image

**Comment by miqo1385** [comment]
it look great in my end. good job.

File: app/category-page/[categoryId]/page.tsx
Comment (gitbiruk2010) [inline app/category-page/[categoryId]/page.tsx:1]: Looks a greate addition for the products to be categorized together. Code is clear  and well organized.

**Review by keiffer213 — APPROVED** [review]
Thanks for getting this done! It looks so much nicer having the tile's display simpler information in category pages!

image

**Review by intisarosman1 — CHANGES_REQUESTED** [review]
I noticed that now the relevant details are also removed, so sizing for all products regardless of category are no longer shown when they should be (shoes should show shoe size, pants should show pant waist size and inseam size, etc.). I can only see the gender and description now.

Screenshot 2024-06-14 012514

**Comment by bcko** [comment]
@Diego-Cano Can you address this and resolve conflicts? I am going to close this pull request. Please reopen or create a new branch (whichever one is easier) when you are ready

> I noticed that now the relevant details are also removed, so sizing for all products regardless of category are no longer shown when they should be (shoes should show shoe size, pants should show pant waist size and inseam size, etc.). I can only see the gender and description now.
> 
> Screenshot 2024-06-14 012514

**Pull request closed without merging**